qa/automation/ScoringPortal.Tests.Scenarios/UsoltsevaL/Sign_in.feature

```diff
@@ -0,0 +1,184 @@
+Feature: 1.1.1.1. sign in with user's Google Account
+
+Background:
+    Given user has a Google Account accountfortesting@gmail.com with assigned phone number +79658905735, password is 0987Po!$
+    And https://qa.ddso-spot.quantori.com/ is opened in browser
+
+@tc:1
+Scenario Outline: 1.1.1.1_01 Logging to the application with valid Email (Google Account has not been logged in)
+    Given user didn't sign in Google Account in page https://accounts.google.com using <browser>
+    When user clicks on "Sign in with Google " button
+    And user fills "accountfortesting@gmail.com" in "Email or phone" text field
+    And user clicks "Next" button
+	And user fills in "0987Po!$" in "Enter your password" text field
```

Comment:
I believe here is an extra tabulation

Reply:
Thank's. Corrected.

Reply:
And there is the same problem in every TC csenario. Could you please take a look?

Reply:
Done